docs/content/theme/pfe-theme-slots.md

```diff
@@ -269,23 +269,23 @@ Theme variables exist so that when a user changes a system property such as colo
 
 Several functions exist in the `pfe-sass` component to make it easier to theme individual components you are building!
     
-1. **Color**:  Rather than using only Sass variables `$red` or hexidecimal colors like `#c00`, please use the `pfe-color()` function along with a theme variable, i.e. `pfe-color(ui-link)`. Occasionally you may have to wrap interpolation syntax `#{ }` around the function to allow Sass to compile, i.e. `#{pfe-color(ui-link)}`. 
+1. **Color**:  Rather than using only Sass variables `$red` or hexidecimal colors like `#c00`, please use the `pfe-color()` function along with a theme variable, i.e. `pfe-color(link)`. Occasionally you may have to wrap interpolation syntax `#{ }` around the function to allow Sass to compile, i.e. `#{pfe-color(link)}`. 
     
 
-    * This function does some heavy-lifting by looking up the `$pfe-colors: ()` map and returning namespaced CSS variables for the [broadcasted color](#broadcasted) (should it be passed down from a dark container), the theme color, and then a fallback color in that order: 
+    * This function does some heavy-lifting by looking up the `$pfe-colors: ()` map and returning namespaced CSS variables for the theme and fallback color, in that order: 
 
 
     ```
     :host {
-       color: pfe-color(ui-link);
+       color: pfe-color(link);
```

Comment:
One more idea for the "best practices" bank -- should we recommend that any value definitions in a component always use `pfe-local()` instead of `pfe-var` or `pfe-color`, even if that variable is not defined?

For example in the CTA, we do not have `FontFamily` defined in the `$variables` map, and yet in the slotted styles we have: 
 
```scss
::slotted(*) {
    font-family:     #{pfe-local($cssvar: FontFamily, $fallback: #{pfe-var(font-family--heading)})};
}
```
This means that if someone wanted to change the font family for all the CTAs on the page, they could do it. But our CTAs just lean on the theme variable for font family. Ideally they match, in this case our CTAs use the heading font but normally you could just keep it simple: 

```scss
::slotted(*) {
    font-family:     #{ pfe-local(FontFamily) };
}
```

Reply:
I'm not going to add any documentation related to this in this PR because it's already large and we need to get this merged in.